seqr/utils/logging_utils.py

```diff
@@ -41,7 +42,7 @@ def formatMessage(self, record):
 
 class SeqrLogger(object):
 
-    def __init__(self, name=None):
+    def __init__(self, name: Optional[str]=None) -> None:
```

Comment:
According to PEP8, you need to add spaces around the equals signs now that you have added annotations: https://peps.python.org/pep-0008/#other-recommendations

> Don’t use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter:
When combining an argument annotation with a default value, however, do use spaces around the = sign